app/views/orga/teams/_form.html.slim

```diff
@@ -77,14 +77,26 @@
     = f.input :season_id, as: :select, collection: Season.order('name DESC') { |e| [s.name, s.id] }
 
   .page-header
-    h2 Conference preferences
+    h2 Conferences
+    h3 Conference preferences
 
   fieldset.conferences_fieldset
     = f.simple_fields_for :conference_preferences, f.object.conference_preferences.sort{ |a,b| a.option <=> b.option} do |c|
       = c.hidden_field :option, value: c.object.option
       = c.input :conference_id, collection: @conferences.group_by(&:region), as: :grouped_select, group_method: :last, label: c.object.option == 1 ? "Primary choice" : "Secondary Choice", prompt: "Conference", wrapper_html: { class: 'col-md-8' }
 
+  .page-header
+    h3 Conference offers
```

Comment:
Are you referring to conference preferences? because the conference offer just appears in this specific view. But that is true, a lot better to share partials. I will do it.